pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -0,0 +1,16 @@
+%div.teacher-announce
+  %a.announce-heading-link{:href=>"http://teacherblog.code.org/"}
+    %div.announce-heading
+      =I18n.t(:dashboard_announcements)
+      %div.heading-view-all
+        =I18n.t(:dashboard_announce_view_all)
+        >>
+  %div.announce-content
+    %div.content-img-container
+      %img.content-img{:src=>"/images/fit-x108/kids4cropped_2x.jpg"}
```

Comment:
Very interesting.  Check out the relevant code in router.rb.. it appears that if the filename has _2x then it knows it's a retina source image and it doubles the returned resolution accordingly.  If you requested kids4cropped.jpg it would still use the _2x source image but return 108 pixels.